packages/meso-js/test/createPostMessageBus.test.ts

```diff
@@ -36,6 +36,20 @@ describe("createPostMessageBus", () => {
     });
   });
 
+  describe("in webview", () => {
+    test("returns a post message bus instance for valid target origin", () => {
+      const fakeWindow = {} as Window;
+      expect(createPostMessageBus(PARTNER_APP_ORIGIN, fakeWindow))
+        .toMatchInlineSnapshot(`
+      {
+        "destroy": [Function],
+        "emit": [Function],
+        "on": [Function],
+      }
+    `);
+    });
+  });
```

Comment:
Do we want to add a sad path case where the target origin doesn't match?

Reply:
I'm having a difficult time thinking of how that error case would come up here and/or what we would do. One thing I was thinking is if the `syntheticWindowHandle` didn't quack like a `window`, we could error out, but it would just break anyway and that would be a result of a mis-integration. So I'm tempted to let it ride.

Open to ideas here though!